package.json

```diff
@@ -88,5 +75,56 @@
     "typedoc": "0.17.7",
     "typescript": "^3.7.2",
     "typescript-docs-verifier": "^1.1.2"
+  },
```

Comment:
pulled all the dot files into here to clean things up a bit

Reply:
note also that there are a lot of "new" and existing linter issues left in this repo, so we'll have to make sure we all update our IDE's linter settings to use the stuff in this repo